src/components/GraphicComponent.js

```diff
@@ -0,0 +1,41 @@
+/* Imports -------------------------------------------------------------------*/
+
+import Inferno from 'inferno'; // eslint-disable-line
+import Component from 'inferno-component';
+
+/* Component -----------------------------------------------------------------*/
+
+class GraphicComponent extends Component {
+	constructor(props) {
+		super(props);
+	}
+
+	render() {
+		return (
+			<div className="stripe-1">
+				<div className="network-bg">
+					<div className="container spacing-1x">
+						<div className="row">
+							<div className="col-12 text-center">
+								<h2 className="section-title-light pt-3 mt-5">Performance</h2>
+								<p className="py-4 section-subtitle-light">
+										Lorem ipsum dolor sit amet, consectetur<br hidden-sm-down /> adipisicing elit,
```

Comment:
Please use the same text that was present in the original readme: https://github.com/fed135/Kalm#performance-analysis instead of lorem ipsum.